dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/Support/VectorStoreFixture.cs

```diff
@@ -13,12 +13,14 @@ public abstract class VectorStoreFixture : IAsyncLifetime
     public virtual string DefaultDistanceFunction => this.TestStore.DefaultDistanceFunction;
     public virtual string DefaultIndexKind => this.TestStore.DefaultIndexKind;
 
+    public virtual string GetUniqueCollectionName() => Guid.NewGuid().ToString();
+
     public virtual Task InitializeAsync()
         => this.TestStore.ReferenceCountingStartAsync();
 
     public virtual Task DisposeAsync()
         => this.TestStore.ReferenceCountingStopAsync();
 
     public virtual TKey GenerateNextKey<TKey>()
-        => this.TestStore.GenerateKey<TKey>(this._nextKeyValue++);
+        => this.TestStore.GenerateKey<TKey>(Interlocked.Increment(ref this._nextKeyValue));
```

Comment:
Oops!

Reply:
I assume the meaning here is positive, so marking as resolved. Otherwise GH won't let me merge oO

![{21855B52-1016-48F9-AFD6-E16FAF3C5638}](https://github.com/user-attachments/assets/aeee9a0b-5607-4087-a009-b2ea8ddecd79)